### PR TITLE
ignored slack errors in test-content command

### PR DESCRIPTION
**Pull request by yaakovi**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Description
ignore slack errors in the test-content command (we are using it as a best-effort - it shouldn't fail the build)

### CODE REVIEW

**Review by roysagi — APPROVED** [review]

File: demisto_sdk/commands/test_content/TestContentClasses.py [hunk 1]
```diff
@@ -708,45 +708,43 @@ def _load_conf_files(conf_path, secret_conf_path):
...
-        for user in user_list:
-            profile = user.get('profile', {})
-            name = profile.get('real_name_normalized', '')
-            if name == user_name:
-                user_id = user.get('id', '')
+        try:
+            if os.environ.get('GITLAB_USER_LOGIN'):
+                user_name = os.environ['GITLAB_USER_LOGIN']
+            else:
+                user_name = self._get_user_name_from_circle()
```
Comment (avidan-H) [inline demisto_sdk/commands/test_content/TestContentClasses.py:737]:
```suggestion
            user_name = os.getenv('GITLAB_USER_LOGIN') or self._get_user_name_from_circle()
```

**Review by avidan-H — CHANGES_REQUESTED** [review]
Shouldn't we just catch the specific exception?

**Comment by yaakovi** [comment]
> Shouldn't we just catch the specific exception?

I would like to ignore any kind of error in this flow as it is not mandatory for the rest of the command
WDYT?

**Comment by avidan-H** [comment]
> > Shouldn't we just catch the specific exception?
> 
> I would like to ignore any kind of error in this flow as it is not mandatory for the rest of the command
> WDYT?

👍🏼  I guess in that case though, not sure if the debug message makes sense since it's assuming what the error is that is being caught and it could very well be something else...

**Comment by yaakovi** [comment]
> > > Shouldn't we just catch the specific exception?
> > 
> > 
> > I would like to ignore any kind of error in this flow as it is not mandatory for the rest of the command
> > WDYT?
> 
> 👍🏼 I guess in that case though, not sure if the debug message makes sense since it's assuming what the error is that is being caught and it could very well be something else...

the function name is `retrieve slack user id` so the message is relevant for the whole block in my opinion.
it may fail to grab the GitHub user but it also means that we can't find the slack user...

**Review by avidan-H — APPROVED** [review]

**Pull request merged**